ballerina-tests/tests/38_bidi_streaming_with_caller_service.bal

```diff
@@ -27,7 +27,7 @@ service "Chat38" on new grpc:Listener(9128) {
         log:printInfo("Invoke the chat RPC");
         int i = 0;
         // Read and process each message in the client stream.
-        check clientStream.forEach(function(ChatMessage38 value) {
+        checkpanic clientStream.forEach(function(ChatMessage38 value) {
```

Comment:
```suggestion
        check clientStream.forEach(function(ChatMessage38 value) {
```

Reply:
IMO, it is okay to return this error.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/2ef479d6607ef5aa5336331221c12c698cdd94cb